dist/placeholder-buster.txt

```diff
@@ -404,7 +407,9 @@ mythicscribes.com##aside[id^="ai_widget-"]
 itstillworks.com##div.whitecard:has(script[src*="/pagead/"])
 
 # https://github.com/NanoAdblockerLab/NanoContrib/pull/11#issuecomment-463024467
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/38
 ||m.majorgeeks.com/mg/mg.html$subdocument
```

Comment:
Is this still needed?

Reply:
A quick test of that entry's example link, tells me that it's seemingly not needed anymore.